fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/CascadesCostModel.java

```diff
@@ -20,417 +20,13 @@
 
 package com.apple.foundationdb.record.query.plan.cascades;
 
-import com.apple.foundationdb.annotation.API;
```

Comment:
Most of this code is now in `PlanningCostModel` as it is only applicable for the planning phase.